pypit/arlris.py

```diff
@@ -160,8 +160,6 @@ def read_lris(raw_file, det=None, TRIM=False):
             # Data section
             section = '[{:d}:{:d},{:d}:{:d}]'.format(preline,nydata-postline, xs, xe)  # Eliminate lines
             dsec.append(section)
```

Comment:
That's right, ampsec is kind of redundant, so I'm removing all instances throughout the code. This was prompted by @bpholden. The new kast red detector doesn't butt together the datasec and oscansec. ampsec assumes that datasec and oscansec are adjacent... and ampsec is not used elsewhere in the code  :-)